packages/payload/src/errors/UnverifiedEmail.ts

```diff
@@ -0,0 +1,15 @@
+import type { TFunction } from '@payloadcms/translations'
+
+import { en } from '@payloadcms/translations/languages/en'
+import { status as httpStatus } from 'http-status'
+
+import { APIError } from './APIError.js'
+
+export class UnverifiedEmail extends APIError {
+  constructor(t?: TFunction) {
```

Comment:
In v4 I would love to adjust the shape of arguments. Can we switch this to an object? It will make it a lot easier if we wanted to allow customization to the status (for example) instead of needing to pass n number of args.